x-pack/legacy/plugins/epm/common/types.ts

```diff
@@ -126,6 +126,11 @@ export interface InstallationAttributes extends SavedObjectAttributes {
   installed: AssetReference[];
 }
 
+export type Datasource = SavedObject<DatasourceAttributes>;
```

Comment:
I assume long term this will become https://github.com/elastic/kibana/blob/feature-ingest/x-pack/legacy/plugins/epm/common/api_specs/ingest/types.ts#L30 ?

Reply:
Yes. For now the datasource saved object looks exactly like the package saved object, this is just   to get started.